miniapps/plasma/cold_plasma_dielectric_solver.cpp

```diff
@@ -582,11 +583,17 @@ CPDSolver::CPDSolver(ParMesh & pmesh, int order, double omega,
    */
    // Bilinear Forms
    a1_ = new ParSesquilinearForm(HCurlFESpace_, conv_);
+   if (pa_)
+   {
+      a1_->real().SetAssemblyLevel(AssemblyLevel::PARTIAL);
+      a1_->imag().SetAssemblyLevel(AssemblyLevel::PARTIAL);
+   }
    a1_->AddDomainIntegrator(new CurlCurlIntegrator(*muInvCoef_), NULL);
    a1_->AddDomainIntegrator(new VectorFEMassIntegrator(*massReCoef_),
                             new VectorFEMassIntegrator(*massImCoef_));
-   if ( kCoef_)
+   if ( kCoef_ )
    {
+      if (pa_) { cout << "WARNING: kCoef_ domain integrator (PA)" << endl; }
```

Comment:
I'm not sure I understand these warning messages.  Are these integrators not supported with PA or not sufficiently tested, or something else?

Reply:
I should've been more specific with the warnings here (initially, I set them up just to notify myself during sample runs if these branches are executed).

I haven't tested these with PA but at least some of these integrators aren't supported with PA yet (`MixedCrossCurlIntegrator`, `MixedWeakCurlCrossIntegrator`). Would you prefer in this case to return or abort with a warning or error message (i.e. a more descriptive one)?

Reply:
Let's abort with a more descriptive error message.